Add rules for Proguard (#98)
uploadcare/uploadcare-android: CHANGELOG.md

    [malware protection](https://uploadcare.com/docs/security/malware-protection/),
    and [background removal](https://uploadcare.com/docs/remove-bg/).
  - Fixed authorization signature for requests with url parameters.

>   - Added keep DTO classes to Proguard's rules, as network JSON Adapter uses reflection on them.

Reviewer: ```suggestion
  - Added Proguard rule to keep DTO classes to avoid issues with network JSON deserialization.
```

- Widget:
  - SocialApi doesn't use `GET /sources` method anymore.

>   - Added keep DTO classes to Proguard's rules, as network JSON Adapter uses reflection on them.

Reviewer: ```suggestion
  - Added Proguard rule to keep DTO classes to avoid issues with network JSON deserialization.
```
